Examples/Scripts/PreFreeSurferPipelineBatch.sh

```diff
@@ -252,9 +252,18 @@ main()
 		#     Average any repeats and use Spin Echo Field Maps for readout
 		#     distortion correction
 		#
-		#   "GeneralElectricFieldMap"
-		#     Average any repeats and use General Electric specific Gradient
-		#     Echo Field Map for readout distortion correction
+		#   "GEHealthCareLegacyFieldMap"
+		#     Average any repeats and use GE HeathCare Legacy specific Gradient
```

Comment:
Repeats of what?

Reply:
This is the averaging of any repeats of the magnitude. 
Such as we have consistent behavior and possibilities for all manfacturer. 
(I has also reused the exact same wording)